spark/v3.3/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestMigrateTableProcedure.java

```diff
@@ -122,6 +122,38 @@ public void testMigrateWithDropBackup() throws IOException {
     Assert.assertFalse(spark.catalog().tableExists(tableName + "_BACKUP_"));
   }
 
+  @Test
+  public void testMigrateWithBackupSuffix() throws IOException {
+    Assume.assumeTrue(catalogName.equals("spark_catalog"));
+    String backupSuffix = "_tmp";
+    String location = temp.newFolder().toString();
+    sql(
+        "CREATE TABLE %s (id bigint NOT NULL, data string) USING parquet LOCATION '%s'",
+        tableName, location);
+    sql("INSERT INTO TABLE %s VALUES (1, 'a')", tableName);
+
+    Object result =
+        scalarSql(
+            "CALL %s.system.migrate(table => '%s', backup_suffix => '%s')",
```

Comment:
Maybe you could add a separate test that tests the following
"CALL %s.system.migrate(table => '%s', drop_backup => true, backup_suffix => '%s')"

Wouldn't hurt to test this combination as the code may change in the future and method chaining doesn't work as expected. it might be better to add a unit test to ensure this doesn't cause any issues. What do you think?

Reply:
Agreed. I thought about that case but didn't add a unit test. Thanks for catching it.